src/ioctls/system.rs

```diff
@@ -9,6 +9,8 @@ use std::os::unix::io::{AsRawFd, FromRawFd, RawFd};
 use cap::Cap;
 use ioctls::vec_with_array_field;
 use ioctls::vm::{new_vmfd, VmFd};
+use ioctls::Result;
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
 use ioctls::{CpuId, Result};
```

Comment:
I think this line needs to be changed to `use ioctls::CpuId;`

Reply:
Yeah, that was a miss. Corrected it.